proposals/nnnn-configurable-time-before-shutdown.md

```diff
@@ -0,0 +1,77 @@
+# Configurable time before shutdown
+
+* Proposal: [SDL-NNNN](nnnn-configurable-time-before-shutdown.md)
+* Author: [Alexander Kutsan](https://github.com/LuxoftAKutsan)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core]
+
+## Introduction
+
+To prevent missing logs after SDL shutdown, additional ini file options should be added : 
+ - Write all logs to file system before shutdown 
+ - Maximum time of SDL shutting down
+ 
+## Motivation
+
+SDL produces a tone of logs in the asynchronous mode in some use cases (like video streaming or big put file),
```

Comment:
Please update to "SDL produces a ton of logs" - also, are you able to include an approximate number here?  Something more specific might be helpful.